src/pudl/metadata/fields.py

```diff
@@ -60,6 +60,147 @@
+        ),
+        "unit": "Mcf",
+        "type": "number",
+    },
+    "disposition_distribution_companies_mcf": {
+        "description": (
+            "Disposition to distribution companies within the report state that does not fall into one of the other reported categories in lines 10.1-17.0. Reference conditions for measurement are 14.73 psia and 60° Fahrenheit."
+        ),
+        "unit": "Mcf",
+        "type": "number",
+    },
+    "disposition_other_pipelines_mcf": {
+        "description": (
+            "Disposition to other pipelines within the report state that does not fall into one of the other reported categories in lines 10.1-17.0. Reference conditions for measurement are 14.73 psia and 60° Fahrenheit."
+            "(14.73 psia and 60° Fahrenheit)"
+        ),
+        "unit": "Mcf",
+        "type": "number",
+    },
+    "disposition_storage_operators_mcf": {
+        "description": (
+            "Disposition to storage operators within the report state that does not fall into one of the other reported categories in lines 10.1-17.0. Reference conditions for measurement are 14.73 psia and 60° Fahrenheit."
+        ),
+        "unit": "Mcf",
+        "type": "number",
+    },
+    "total_disposition_mcf": {
+        "description": (
+            "Total disposition within the report state, as reported by the operator. "
+            "This includes disposition to consumers which is reported in core_eia176__yearly_gas_disposition_by_consumer. "
+            "Note that the reported total disposition and the sum of values in this table and core_eia176__yearly_gas_disposition_by_consumer often don't match as it would be expected. "
+            "Reference conditions for measurement are 14.73 psia and 60° Fahrenheit."
+        ),
+        "unit": "Mcf",
+        "type": "number",
+    },
+    "unaccounted_for_mcf": {
+        "description": (
+            "The difference between gas supply and disposition. A positive entry indicates supply in excess of accounted-for disposition. A negative entry indicates accounted-for disposition exceeds reported supply. This is calculated as the difference between Part 4 Line 7.0 and Part 6 Line 19.0, and is reported as Line 20.0 in the original form."
+        ),
+        "unit": "Mcf",
+        "type": "number",
+    },
+    "disposition_out_of_state_mcf": {
+        "description": (
+            "Total volume of the operator's deliveries across or to state lines or U.S. borders. This has been summed from the detailed data reported by each company on Line 14.0 of the original form in order to preserve the primary key of the table. Reference conditions for measurement are 14.73 psia and 60° Fahrenheit."
```

Comment:
Do we anticipate that detailed breakdown also being available in its own separate table as well?